R/DataProcessing.R

```diff
@@ -905,7 +909,11 @@ add_qfeatures <- function(dataList, qfeatures, fileAnnotation = NULL, siriusCate
   #adding HEX color codes from external annotations to the annotationColorsMapInitValue of dataFrameHeader
   
   # Copy the selected column by user, Remove duplicates and exclude the first row
-  uniqueAnnotations0 <- unique(unlist(strsplit(metaboliteProfile$Annotation, ",")))
+  
+  ## gp: This line was causing issues because some annotations contain commas on purpose.
+  ## Not clear is which cases the strsplit was needed
+  # uniqueAnnotations0 <- unique(unlist(strsplit(metaboliteProfile$Annotation, ",")))
```

Comment:
@NorStorz , any idea why you used the strsplit here ? We remove it for now, but if there was a reason we need to add a test for several cases and make sure it works for all of them.